src/main/java/com/labsynch/labseer/domain/Project.java

```diff
@@ -0,0 +1,35 @@
+package com.labsynch.labseer.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+
+import javax.validation.constraints.Size;
+
+import org.springframework.roo.addon.javabean.RooJavaBean;
+import org.springframework.roo.addon.jpa.activerecord.RooJpaActiveRecord;
+import org.springframework.roo.addon.json.RooJson;
+import org.springframework.roo.addon.tostring.RooToString;
+
+@RooJavaBean
+@RooToString
+@RooJpaActiveRecord(finders = { "findProjectsByCodeEquals", "findProjectsByNameEquals" })
+@RooJson
+public class Project {
```

Comment:
Will be removing in a commit shortly.  While removing project, I found places it was still being used such as in the search form in Creg.  I am modifying creg search to use the same mechanism of passing in allowed projects as part of the request so that roo can filter.